crates/rpc/rpc-types/src/eth/transaction/typed.rs

```diff
@@ -112,3 +112,12 @@ impl Decodable for TransactionKind {
         }
     }
 }
+
+impl From<TransactionKind> for reth_primitives::TransactionKind {
```

Comment:
@mattsse I assume this type won't be necessary in the future and we'll be using it from Revm? here we implemented it again just for the RLP methods?